end-to-end-test/remote/specs/core/studyview.spec.js

```diff
@@ -49,7 +49,7 @@ const STUDY_SUMMARY_RAW_DATA_DOWNLOAD =
 const CNA_GENES_TABLE = "[data-test='copy number alterations-table']";
 const CANCER_GENE_FILTER_ICON = "[data-test='header-filter-icon']";
 
-const WAIT_FOR_VISIBLE_TIMEOUT = 30000;
+const WAIT_FOR_VISIBLE_TIMEOUT = 10000;
```

Comment:
should this have been lowered?

Reply:
ahh... lowered the wrong one... I will change this back.